src/prog_models/visualize.py

```diff
@@ -97,11 +102,13 @@ def set_plot_options(opt: dict) -> dict:
     opt : dict
         dict of plot options with default values added.
 
-    Example
+    Example:
```

Comment:
```suggestion
    Example
```

Sphinx expects either the line afterwards or the :